tsconfig.json

```diff
@@ -6,7 +6,9 @@
     "rootDir": "./src",
     "strict": true,
     "noImplicitAny": true,
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "sourceMap": true,
+    "inlineSourceMap": false
```

Comment:
If source maps are needed for debugging, consider using `inlineSourceMap: true` instead of separate source map files. This can simplify deployment and prevent issues with missing source map files.